jwst/resample/resample_step.py

```diff
@@ -26,13 +26,14 @@ class ResampleStep(Step):
     """
 
     spec = """
-        pixfrac = float(default=None)
-        kernel = string(default=None)
-        fillval = string(default=None)
-        weight_type = option('exptime', default=None)
+        pixfrac = float(default=1.0)
+        kernel = string(default='square')
+        fillval = string(default='INDEF')
```

Comment:
I think `fillval` should remain `None` here.  See #2219

Reply:
Clarification: The original CFG file had `fillval=INDEF`, Would setting the default to `None` now without #2219 being dealt with, be OK?

Reply:
Offline conversation says this is ok to leave for now.